### PR TITLE
Update zh-Hant.html

### PR DESCRIPTION
**Pull request by Oschangkai**
Modify some translation

### CODE REVIEW

**Comment by squidfunk** [comment]
Could you please provide a detailed explanation of what you changed and why? Also can somebody with a Chinese background review this?

**Comment by Oschangkai** [comment]
I changed some terms which are Simplified Chinese originally to Traditional Chinese.

**Comment by jfcherng** [comment]
@squidfunk This PR basically modifies the translation to fit Taiwanese habit.
Technically speaking, zh-Hant means Traditional Chinese rather than involving regional habit.
But IIRC, you insisted that the file name should be `zh-Hant` rather than `zh-TW` or something else.

File: material/partials/language/zh-Hant.html [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-  "search.result.placeholder": "鍵入以開始檢索",
+  "search.result.placeholder": "輸入文字後開始檢索",
   "search.result.none": "沒有找到符合條件的結果",
-  "search.result.one": "找到 1 个符合條件的結果",
+  "search.result.one": "找到 1 個符合條件的結果",
   "search.result.other": "# 個符合條件的結果",
   "search.tokenizer": "[\，\。]+",
   "skip.link.title": "跳轉至",
-  "source.link.title": "前往 Github 倉庫",
+  "source.link.title": "前往 Github Repo",
```
Comment (jfcherng) [inline material/partials/language/zh-Hant.html]: I would claim this modification to be a personal habit.

**Comment by squidfunk** [comment]
@jfcherng thank you for your assessment!

I would also favor a translation without regional habits, so I will close this PR. If you want to provide `zh-TW` feel free to open a new PR with the respective translations.

**Pull request closed without merging**